manager/volume.go

```diff
@@ -471,23 +472,18 @@ func (m *VolumeManager) Activate(volumeName string, frontend string) (v *longhor
 		return nil, err
 	}
 
-	var engine *longhorn.Engine
 	es, err := m.ds.ListVolumeEngines(v.Name)
 	if err != nil {
 		return nil, fmt.Errorf("failed to list engines for volume %v: %v", v.Name, err)
 	}
 	if len(es) != 1 {
 		return nil, fmt.Errorf("found more than 1 engines for volume %v", v.Name)
 	}
-	for _, e := range es {
-		engine = e
-	}
 
-	if v.Status.LastBackup != engine.Status.LastRestoredBackup || engine.Spec.RequestedBackupRestore != engine.Status.LastRestoredBackup {
-		logrus.Infof("Standby volume %v will be activated after finishing restoration, "+
-			"backup volume's latest backup: %v, "+
-			"engine requested backup restore: %v, engine last restored backup: %v",
-			v.Name, v.Status.LastBackup, engine.Spec.RequestedBackupRestore, engine.Status.LastRestoredBackup)
```

Comment:
NIT: Since we don't fetch the engine and check the last restored backup any more,  you can directly remove the whole engine get and check codes here.